components/playlist/browser/resources/components/app.tsx

```diff
@@ -304,7 +302,7 @@ export class PlaylistPage extends React.Component<Props, State> {
   }
 }
 
-export const mapStateToProps = (state: States.ApplicationState) => ({
+export const mapStateToProps = (state: ApplicationState) => ({
```

Comment:
Should we be using a `useSelector` here instead of `mapStateToProps`? (my redux is pretty rusty sorry, I could be wrong here)

Reply:
Yeah, it'd be better. But I'd like to leave this file(old web ui) as is and dump app.v1.tsx one day 😄